apps/nextjs-website/src/components/StripeLink.tsx

```diff
@@ -1,8 +1,7 @@
 'use client';
-import { StripeLink as StripeLinkEC } from '@pagopa/pagopa-editorial-components';
-import { StripeLinkProps } from '@pagopa/pagopa-editorial-components/dist/components/StripeLink';
-import { Stack } from '@mui/material';
 import MarkdownRenderer from './MarkdownRenderer';
```

Comment:
```suggestion
import Icon from './Icon';
import MarkdownRenderer from './MarkdownRenderer';
```
Missing `Icon` import